### ISSUE TITLE
Using JSX?

### ISSUE THREAD

**Issue by MoradiDavijani** [uber/react-map-gl#871]
Hi. Thanks for all the great work.

I'm wondering if there is any reason (or blocking issue) not using JSX in this project?
I think it will be much simpler to understand components sources and contribute when using JSX.

If it's OK I can try it with a pull request.

**Comment by ibgreen** [comment]
Do not have an opinion but I can provide some background:

We used to write all our projects in pure JavaScript (ES6/ES2018) and run code in latest Chrome and test in Node.js without transpilation (better debug experience, no source maps needed). Using JSX would mean having to introduce babel or similar, we have very few React components and benefit of JSX was small)

However, I believe this project has since introduced flow, which while not hard to strip away does diverge from the original "pure JS" setup. Also more React components have been added, increasing the value of JSX.
